### PR TITLE
Font relative paths

### PR DESCRIPTION
**Pull request by djbe**
Fixes https://github.com/SwiftGen/SwiftGen/issues/323.
Refs https://github.com/SwiftGen/SwiftGen/issues/324.

templates related PR: https://github.com/SwiftGen/templates/issues/71

- [x] Align templates to master before merge

### CODE REVIEW

File: Sources/Parsers/FontsParser.swift [hunk 1]
```diff
@@ -84,6 +84,8 @@ public final class FontsParser: Parser {
 
   public func parse(path: Path) {
     let dirChildren = path.iterateChildren(options: [.skipsHiddenFiles, .skipsPackageDescendants])
+	let parentDir = path.absolute().parent()
```
Comment (AliSoftware) [inline Sources/Parsers/FontsParser.swift]: Indentation?

Comment (djbe) [reply Sources/Parsers/FontsParser.swift]: For f. sake... Xode!

**Review by AliSoftware — APPROVED** [review]

**Review by AliSoftware — COMMENTED** [review]
Wait, shouldn't we add some additional tests to ensure we won't have a regression regarding relative/absolute path in the future?

**Comment by djbe** [comment]
What do you want to test? The `relative(to:)` method?

**Comment by AliSoftware** [comment]
Yeah and also try to call the while framework entry-point with a relative path and an absolute path and check they don't leak the full path in either case, something like that?

**Comment by AliSoftware** [comment]
(Also reminder: don't forget to wait for SwiftGen/templates#71 then **undo the `Podfile` changes** before merging this)

**Comment by djbe** [comment]
That was in the templates repo, I already removed the change 👍

**Comment by AliSoftware** [comment]
Haha I was actually searching it after I made that comment and failed to find it again and thought I needed some more sleep (which might still be true anyway 😜 )

**Comment by djbe** [comment]
Todo: align subrepo before merged

**Pull request merged**